go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/alexellis/derek
+
+go 1.15
+
+require (
+	github.com/alexellis/hmac v0.0.0-20180624211220-5c52ab81c0de
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
```

Comment:
incompatible?

Reply:
`+incompatible` means the dependency has a semver major version of 2 or higher and is not a Go module yet (it doesn't have go.mod in its source code).